src/common/exif.cc

```diff
@@ -914,11 +914,11 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     {
       const float focus_position = pos->toFloat();
 
-      if (FIND_EXIF_TAG("Exif.Photo.FocalLengthIn35mmFilm")) {
-          const float focal_length_35mm = pos->toFloat();
+      if (FIND_EXIF_TAG("Exif.Photo.FocalLengthIn35mmFilm") && focus_position) {
```

Comment:
Might be better to reverse the order, i.e. check for `focus_position` first because then the && can be short-circuited if false and won't bother with another (more expensive) exiv2 lookup.

Reply:
Good point. I made this pool request quickly in hopes of making it to the deadline and overlooked this. Thanks, fixed.